### PR TITLE
Add romulus commands.

### PR DESCRIPTION
**Pull request by howbazaar**
Also update dependencies.

### CODE REVIEW

File: cmd/juju/romulus/listplans/list_plans.go [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+	"github.com/gosuri/uitable"
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+	"github.com/juju/juju/cmd/modelcmd"
+	"gopkg.in/macaroon-bakery.v1/httpbakery"
+	"gopkg.in/yaml.v2"
+
+	api "github.com/juju/romulus/api/plan"
+	rcmd "github.com/juju/romulus/cmd"
```
Comment (cmars) [inline cmd/juju/romulus/listplans/list_plans.go]: github.com/juju/juju/cmd/juju/romulus/commands

**Comment by cmars** [comment]
LGTM

**Comment by howbazaar** [comment]
$$merge$$

**Comment by howbazaar** [comment]
$$try-now$$ agree command fails on windows due to using the less command.

**Comment by howbazaar** [comment]
$$typo$$

**Comment by howbazaar** [comment]
$$intermittent-failure$$

**Pull request merged**